network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ThreeWindingsTransformerAdderImpl.java

```diff
@@ -246,12 +246,18 @@ public LegAdder newLeg1() {
 
     @Override
     public LegAdder newLeg2() {
-        return new LegAdderImpl(2, this);
+        LegAdderImpl legAdder = new LegAdderImpl(2, this);
+        legAdder.g = 0.0;
+        legAdder.b = 0.0;
```

Comment:
why not the same for leg1 ?
and so, why having changed the default value from 0.0 to NaN ?

Reply:
because homogenization : see : `powsybl-core/iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ThreeWindingsTransformerAdderImpl.java`